.github/dependabot.yml

```diff
@@ -9,4 +9,5 @@ updates:
     schedule:
       interval: "daily"
     ignore:
-      dependency-name: "org.codehaus.groovy:groovy-all"
+      - dependency-name: "org.codehaus.groovy:groovy-all"
+      - dependency-name: "commons-beanutils:commons-beanutils"
```

Comment:
BTW we should be including comments referring to written reasons for avoiding certain updates. For Groovy it is pretty well-known but for BeanUtils the issues in #4928 and #4328 were pretty technical.

Reply:
just wanted to fix dependabot in this PR,

comments added in https://github.com/jenkinsci/jenkins/pull/5130 feel free to debate / suggest phrasing